variants/dev/5.6.3/variables.json

```diff
@@ -0,0 +1,38 @@
+{
+    "env": "dev",
+    "apache": {
+        "docroot": "/var/www/html"
+    },
+    "shopware": {
+        "version": "5.6.3",
+        "download_url": "https://www.shopware.com/de/Download/redirect/version/sw5/file/install_5.6.3_fec7645a72a0393f8a39f48ddd6c27e138f44513.zip"
+    },
+    "composer": {
+        "version": 1
+    },
+    "php": {
+        "default_version": "7.0",
```

Comment:
pls set this version to 7.2 as 7.0 is not supported in this version, this leads to the build error

Reply:
Is now set to 7.2